lib/urlwatch/cli.py

```diff
@@ -100,7 +100,12 @@ def main():
 
     # setup storage API
     config_storage = YamlConfigStorage(command_config.config)
-    cache_storage = CacheMiniDBStorage(command_config.cache)
+
+    if command_config.cache.startswith('redis://') or command_config.cache.startswith('rediss://'):
```

Comment:
Could be:

    if any(command_config.cache.startswith(prefix) for prefix in ('redis://', 'rediss://')):